docs/support-policy.md

```diff
@@ -0,0 +1,39 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Packages support policy
+
+`tezos-packaging` provides packages for Ubuntu, Fedora and macOS for each stable and release-candidate
+version of the Octez.
+
+This document targets to explain which versions of the aforementioned OSes are targeted
+by our packages.
+
+## Ubuntu packages
+
+We provide support for all LTSes supported by Canonical:
+* 18.04 LTS (Bionic Beaver)
+* 20.04 LTS (Focal Fossa)
+
+Also, we provide support for non-LTS versions on request from the users.
+E.g. 21.04 (Hirsute Hippo) support was requested in [#212](https://github.com/serokell/tezos-packaging/issues/212).
+
+There are packages for `arm64` and `amd64` architectures.
+
+## Fedora packages
+
+We provide support for all maintained Fedora versions:
+* Fedora 34
+* Fedora 35
```

Comment:
LGTM now... except that it looks like, for the time being, we're unable to support Fedora 35 after all :kekw:

Reply:
Uhmm, yes :see_no_evil: